views/maintenance.hbs

```diff
@@ -30,7 +30,7 @@
 </head>
 <body class="aui-page-hybrid">
 <div class="ac-content">
-  <div class="page-content">
+  <div class="pageContent">
```

Comment:
Why this change?  Also, rule of thumb for css classes is to use kebab case, not camel case.

Reply:
As part of the UI updates everything that was shared went into `global.css`. With plain CSS, yes, but when using BEM conventions, you can choose to use camel case. https://en.bem.info/methodology/naming-convention/

Reply:
Is it how it named in global.css? If not, should we rename it in the css file as well? 🤔

Reply:
It's fine for now, I think we'll need to discuss in the future so that we use an agreed upon standard across the board.